src/dvclive/live.py

```diff
@@ -299,6 +307,33 @@ def log_param(
         """Saves the given parameter value to yaml"""
         self.log_params({name: val})
 
+    def log_artifact(
+        self,
+        path: str,
+    ):
+        from dvc.exceptions import OutputDuplicationError
+
+        """Saves a local file or directory as an artifact"""
+        if not isinstance(path, str):
+            raise InvalidDataTypeError("artifact", type(path))
+
+        if self._dvc_repo is not None:
+            if path in self._outs or self._inside_dvc_exp:
```

Comment:
If we are in `self._inside_dvc_exp` and this path was never added before, this will fail with `dvc.exceptions.NoOutputOrStageError`, right? 

We should also check that `path` is already tracked by DVC.